libvirt/tests/src/guest_os_booting/os_configuration/without_default_os_attributes.py

```
@@ -20,10 +75,16 @@ def run(test, params, env):
     3) Check the guest dumpxml.
     """
     vm_name = guest_os.get_vm(params)
-    firmware_type = params.get("firmware_type")
     os_dict = eval(params.get("os_dict"))
```

Ruff (0.15.12)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)